hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassCluster.java

```diff
@@ -144,6 +149,13 @@ public static synchronized Session getSession(boolean overwrite) throws Exceptio
             String nodes = AlertProperties.getProperty(ALERTS_CASSANDRA_NODES, ALERTS_CASSANDRA_NODES_ENV, "127.0.0.1");
             int attempts = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_RETRY_ATTEMPTS, "5"));
             int timeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_RETRY_TIMEOUT, "2000"));
+            /** @see SocketOptions.DEFAULT_CONNECT_TIMEOUT_MILLIS */
+            int connTimeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_CONNECT_TIMEOUT,
+                    ALERTS_CASSANDRA_CONNECT_TIMEOUT_ENV, "5000"));
+            /** @see SocketOptions.DEFAULT_READ_TIMEOUT_MILLIS */
+            int readTimeout = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_READ_TIMEOUT,
+                    ALERTS_CASSANDRA_READ_TIMEOUT_ENV, "12000"));
```

Comment:
OK, as we discussed in IRC, let's not override (keep the datastax OOB settings) unless our sysprops have been set.  After that this looks good to me.